src/actions/dropbox/dropbox.ts

```diff
@@ -0,0 +1,211 @@
+import * as querystring from "querystring"
+import * as https from "request-promise-native"
+import {URL} from "url"
+
+import Dropbox = require("dropbox")
+import * as winston from "winston"
+import * as Hub from "../../hub"
+
+export class DropboxAction extends Hub.OAuthAction {
+    name = "dropbox"
+    label = "Dropbox"
+    iconName = "dropbox/dropbox.png"
+    description = "Send query results directly to a file in your Dropbox."
+    supportedActionTypes = [Hub.ActionType.Query, Hub.ActionType.Dashboard]
+    usesStreaming = false
+    minimumSupportedLookerVersion = "6.4.0"
+    requiredFields = []
+    params = [
+      {
+        description: "Dropbox Application Key",
```

Comment:
I'm still going to be a pest about trying to make it so that users don't need to provide this key/secret by having us own the Dropbox app.

Reply:
@phillipperalez is there a need for the user to create their own dropbox app? it seems simpler for a single action hub dropbox app to be created.

Reply:
I need to talk with dropbox. It's a relatively easy change for me to make if they can just make us a global app but that isn't always a guarantee